mk/docs.mk

```diff
@@ -37,12 +37,31 @@ docs/generated/openapi.yaml:
 	mkdir -p $(dir $@)
 	mkdir -p $(OAPI_TMP_DIR)/policies
 	mkdir -p $(OAPI_TMP_DIR)/resources
+
+	for resource in Mesh MeshGateway; do \
+  		lowercase=$$(echo $${resource} | tr '[:upper:]' '[:lower:]'); \
+		mkdir -p $(OAPI_TMP_DIR)/resources/$${lowercase}/; \
+		go run tools/template/main.go --template tools/policy-gen/templates/schema.yaml --data '{"Scope": "Global"}' | \
+		$(YQ) -o json | \
+		$(JQ) --slurpfile defs ./docs/generated/raw/protos/$${resource}.json -f tools/template/resolve_refs.jq | \
```

Comment:
YQ works over json (because json is a subset of yaml), is there a specific reason to use jq here?

Reply:
yq doesn't have `walk` function which is needed to resolve `$ref`s.